core/parentchain/light-client/src/io.rs

```diff
@@ -80,6 +80,11 @@ where
 
 	if genesis_hash == params.genesis_header.hash() {
 		validator.set_state(validation_state);
+		// The init_parachain_validator function clear the state every time,
```

Comment:
Minor thing, here it is the `init_grandpa_validator` :)

Reply:
Yes, sorry!